src/sc/embedder/content/embedder.py

```diff
@@ -438,3 +439,33 @@ class EmbedderVideoJS(BrowserView):
 
     def __call__(self):
         return self.index()
+
+
+@indexer(IEmbedder)
+def searchable_text_indexer(obj):
+    """SearchableText should contain id, title, description, body text,
+    alternate text and keywords.
+    """
+    transformer = ITransformer(obj)
+    text = obj.text
+    if IRichTextValue.providedBy(text):
+        text = transformer(text, 'text/plain')
+    else:
+        text = ''
```

Comment:
if `text` is not a RichText will always be empty.. well.. `text` [will be always be RichText](https://github.com/simplesconsultoria/sc.embedder/blob/master/src/sc/embedder/content/embedder.py#L137-L140) then you can change it to:
```python
    alternate_text = transformer(obj.text, 'text/plain')
```

Reply:
`text` could be None.